src/plugins/intel_gpu/include/intel_gpu/primitives/average_unpooling.hpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
```

Comment:
This file is removed in the previous commit.

Reply:
Removed it.